gr-picoscope/picoscope.hpp

```diff
@@ -0,0 +1,559 @@
+#ifndef GR_PICOSCOPE_PICOSCOPE_HPP
+#define GR_PICOSCOPE_PICOSCOPE_HPP
+
+#include <node.hpp>
+
+#include <fmt/format.h>
+
+#include <functional>
+
+// #define GR_PICOSCOPE_POLLER_THREAD 1
+
+namespace gr::picoscope {
```

Comment:
`namespace fair::picoscope` ... this will not be an official GNU Radio but OOT module.